<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFlowDialog</class>
 <widget class="QDialog" name="AddFlowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>368</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>文泉驿微米黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>添加流表</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>15</y>
     <width>624</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="5" colspan="2">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="DPIDLineEdit">
        <property name="text">
         <string>&quot;00:00:00:00:00:00:00:01&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="flowNameLineEdit">
        <property name="text">
         <string>&quot;flow-1&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inPortLineEdit">
        <property name="text">
         <string>&quot;1&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ethTypeLineEdit">
        <property name="text">
         <string>&quot;0x0800&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="srcMACLineEdit">
        <property name="text">
         <string>&quot;00:0c:29:94:3d:93&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="desMACLineEdit">
        <property name="text">
         <string>&quot;00:0c:29:94:3d:90&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="priorityLineEdit">
        <property name="text">
         <string>&quot;32768&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="activeComboBox">
        <item>
         <property name="text">
          <string>true</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>false</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="actionLineEdit">
        <property name="text">
         <string>&quot;output=2&quot;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="7">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="6" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="srcIPLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="desIPLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="IPprotoLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="IPTosLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="srcPortLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dstPortLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="vlanIDLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="vlanIDPriorityLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="4" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>源IP：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>目的IP：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>IP规则：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>IP Tos：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>源端口：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>目的端口：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>Vlan ID：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>Vlan ID priority：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>交换机DPID：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
          <stylestrategy>PreferDefault</stylestrategy>
         </font>
        </property>
        <property name="text">
         <string>流表名称：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>输入端口：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>以太网类型：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>源MAC：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>目的MAC：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>优先级：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>生效：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>操作：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addAgainButton">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>继续添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="submitButton">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>提交</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3" rowspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>28</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
